config/src/lib.rs

```diff
@@ -784,21 +791,57 @@ impl Config {
                 return match path.is_file() {
                     true => Some(path.to_path_buf()),
                     false => None,
-                }
+                };
             }
             let cwd = std::env::current_dir().ok()?;
             let mut cwd = cwd.as_path();
             loop {
                 let file_path = cwd.join(path);
                 if file_path.is_file() {
-                    return Some(file_path)
+                    return Some(file_path);
```

Comment:
fyi we use nightly formatting `cargo +nightly fmt`

Reply:
removed fmt patches from commit